src/kernel.cpp

```diff
@@ -134,6 +126,8 @@ cl_int cvk_kernel::set_arg(cl_uint index, size_t size, const void* value) {
     return ret;
 }
 
+bool cvk_kernel::args_valid() const { return m_argument_values->args_valid(); }
```

Comment:
That could have been in the header but we've iterated enough on that change now :).